src/injected/content/clipboard.js

```diff
@@ -3,10 +3,13 @@ import { sendCmd } from '../utils';
 import { addEventListener, logging } from '../utils/helpers';
 import bridge from './bridge';
 
-const { execCommand } = Document.prototype;
+// some old versions of Firefox define it on __proto__
+// eslint-disable-next-line no-proto
+const execCommand = Document.prototype.execCommand || document.__proto__.execCommand;
```

Comment:
```
document
  |
HTMLDocument - (Firefox ESR) execCommand
  |
Document - (Chrome and newer Firefox) execCommand
```

So we can just read `execCommand` from `HTMLDocument.prototype` or even `document` since `document` cannot be modified at `document-start`.

Reply:
Yeah, good idea.